Description: Modified command and rules to use Debian specific tools
 By default dwm uses uxterm as terminal emulator
 this is modified now to use x-terminal-emulator
 command instead.
 .
 Also patch modifies class rules to use Iceweasel instead of Firefox
 as that is the replacement for Firefox on Debian system.
Author: Vasudev Kamath <kamathvasudev@gmail.com>
Forwarded: not-needed
Last-Update: 2013-09-15
--- a/config.def.h
+++ b/config.def.h
@@ -18,8 +18,8 @@
 
 static const Rule rules[] = {
 	/* class      instance    title       tags mask     isfloating   monitor */
-	{ "Gimp",     NULL,       NULL,       0,            True,        -1 },
-	{ "Firefox",  NULL,       NULL,       1 << 8,       False,       -1 },
+	{ "Gimp",       NULL,       NULL,       0,            True,        -1 },
+	{ "Iceweasel",  NULL,       NULL,       1 << 8,       False,       -1 },
 };
 
 /* layout(s) */
@@ -47,7 +47,7 @@
 
 /* commands */
 static const char *dmenucmd[] = { "dmenu_run", "-fn", font, "-nb", normbgcolor, "-nf", normfgcolor, "-sb", selbgcolor, "-sf", selfgcolor, NULL };
-static const char *termcmd[]  = { "uxterm", NULL };
+static const char *termcmd[]  = { "x-terminal-emulator", NULL };
 
 static Key keys[] = {
 	/* modifier                     key        function        argument */
@@ -102,4 +102,3 @@
 	{ ClkTagBar,            MODKEY,         Button1,        tag,            {0} },
 	{ ClkTagBar,            MODKEY,         Button3,        toggletag,      {0} },
 };
-
